models/us_fdic_bankfind/code/register_metadata.py

```
@@ -0,0 +1,627 @@
+COVERAGE_END = (2026, 6)  # 2026Q2
+
+
+def fn(name: str):
+    f = getattr(server, name)
+    return getattr(f, "fn", f)
+
+
+def lookup(category: str, slug: str, env: str) -> str | None:
+    try:
+        return fn("lookup_id")(category=category, slug=slug, env=env)["id"]
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)